Resources/Data Sets/index_nl.json

```diff
@@ -610,7 +610,7 @@
           "path": "BuildIt_Binary.jasp",
           "description": "BuildIt's niet-monetaire audit populatie bestaande uit 3500 gegevens. <br><br> Het voorbeeldbestand in JASP laat zien hoe binaire audit workflow kan worden toegepast. <br><br> <i>Dataset gebruikt in Derks, de Swart, Wagenmakers, Wille & Wetzels (under review)<\/i>",
           "kind": "file",
-          "associated_datafile": "BuildIt_Binary.csv"
```

Comment:
So just to verify.. both .jasp files (BuildIt_Binary.jasp and BuildIt_Monetary.jasp) should use the same .csv file?

Reply:
Yeah, they are different walkthroughs of the same dataset

Reply:
Alright